walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/HttpAgentUtils.kt

```diff
@@ -18,6 +18,13 @@ internal class HttpAgentUtils @Inject constructor(@Named("userAgentInfo") privat
         UrlFormEncoded
     }
 
+    fun formatPreferValues(values: List<String>): String {
+        // https://www.rfc-editor.org/rfc/rfc7240
+        // Assumes each is a preference with included parameter (preference; parameter="";...)
+        // may comma separate.
+        return values.joinToString(", ")
```

Comment:
I just noticed according to this RFC, it should be `;` not `,` here.

Reply:
This should probably be handled in a separate PR. This bug has already shipped and will require coordination with server-side to fix.